tensorboard/dataclass_compat_test.py

```diff
@@ -253,6 +253,23 @@ def test_graph_def_experimental_filter_graph(self):
 
         self.assertProtoEquals(expected_graph_def, new_graph_def)
 
+    def test_graph_def_experimental_filter_graph_corrupt(self):
+        # Simulate legacy graph event with an unparseable graph
+        old_event = event_pb2.Event()
+        old_event.step = 0
+        old_event.wall_time = 456.75
+        # Careful: some proto parsers choke on byte arrays filled with 0, but
+        # others don't (silently producing an empty proto, I guess).
+        # Thus `old_event.graph_def = bytes(1024)` is an unreliable example.
+        old_event.graph_def = b"bogus"
+
+        new_events = self._migrate_event(
+            old_event, experimental_filter_graph=True
+        )
+        # _migrate_event emits both the original event and the migrated event,
+        # but here there is no migrated event becasue the graph was unparseable.
+        self.assertLen(new_events, 1)
```

Comment:
Also assert that `new_events[0]` is equal to `old_event`.

Reply:
Done.